lib/python/qmk/constants.py

```diff
@@ -63,6 +63,51 @@
     'RESET': 'QK_BOOT'
 }
 
+# Map VID:PID values to bootloaders
+BOOTLOADER_VIDS_PIDS = {
+    'atmel-dfu': {
+        ("03eb", "2fef"),  # ATmega16U2
+        ("03eb", "2ff0"),  # ATmega32U2
+        ("03eb", "2ff3"),  # ATmega16U4
+        ("03eb", "2ff4"),  # ATmega32U4
+        ("03eb", "2ff9"),  # AT90USB64
+        ("03eb", "2ffa"),  # AT90USB162
+        ("03eb", "2ffb")  # AT90USB128
+    },
+    'kiibohd': {("1c11", "b007")},
+    'stm32-dfu': {
+        ("1eaf", "0003"),  # STM32duino
+        ("0483", "df11")  # STM32 DFU
+    },
+    'apm32-dfu': {("314b", "0106")},
+    'bootloadhid': {("16c0", "05df")},
+    'usbasploader': {("16c0", "05dc")},
+    'usbtinyisp': {("1782", "0c9f")},
+    'md-boot': {("03eb", "6124")},
+    'caterina': {
```

Comment:
Cannot verify this personally on the actual hardware, but apparently Atreus 2 comes with a special build of the `caterina` bootloader using its own VID:PID (already present in the current `util/udev/50-qmk.rules`).
```suggestion
    'caterina': {
        # pid.codes shared PID
        ("1209", "9203"),  # Keyboardio Atreus 2 Bootloader
```